alerting/src/main/kotlin/org/opensearch/alerting/TriggerService.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.alerting
+
+import org.apache.logging.log4j.LogManager
+import org.opensearch.alerting.model.Alert
+import org.opensearch.alerting.model.Monitor
+import org.opensearch.alerting.model.Trigger
+import org.opensearch.alerting.model.TriggerRunResult
+import org.opensearch.alerting.script.TriggerExecutionContext
+import org.opensearch.alerting.script.TriggerScript
+import org.opensearch.client.Client
+import org.opensearch.script.ScriptService
+
+/** Service that handles executing Triggers */
+class TriggerService(val client: Client, val scriptService: ScriptService) {
```

Comment:
Are we using `Client` anywhere?

Reply:
Will update this and the address the comment below in a subsequent PR which has the full implementation of the `TriggerService` since some things changed between this commit and the final state.